1-JavaScript/Week3/homework/ex7-mindPrivacy.js

```diff
@@ -28,8 +28,14 @@ const employeeRecords = [
 ];
 
 // ! Function under test
-function filterPrivateData(/* parameter(s) go here */) {
-  // TODO complete this function
+function filterPrivateData(filteredRecords) {
+  const array=[];
+for (const obj of filteredRecords){
```

Comment:
```suggestion
for (const employeeRecord of filteredRecords){
```
`obj` does not say anything about the object you are working with.